source/Ship.cpp

```diff
@@ -1439,18 +1439,18 @@ int Ship::Scan()
 	if(government->IsPlayer() || (target->GetGovernment()->IsPlayer() && activeScanning))
 		Audio::Play(Audio::Get("scan"), Position());
 	
-	if(startedScanning && government->IsPlayer())
+	if(startedScanning && government->IsPlayer() && !target->GetGovernment()->IsPlayer())
```

Comment:
If you look at the line of code following this one, it actually says, "Attempting to scan the ship <name>." It's not one of the messages that mentions the government name.